Allow extends from empty base-class and define empty class. (#3362)
modelica/ModelicaSpecification: chapters/functions.tex

  Since the class is partial, it is not possible to define an instance of this class.
  \end{nonnormative}
\item

>   An external object class shall be directly extended from {\lstinline!ExternalObject!}, shall have exactly two function definitions, called {\lstinline!constructor!} and {\lstinline!destructor!}, may extend empty classes (\cref{empty-class}), but not contain any other elements.

Reviewer: With one exception (that should also be fixed), we write say _extend from_:
```suggestion
  An external object class shall be directly extended from {\lstinline!ExternalObject!}, shall have exactly two function definitions, called {\lstinline!constructor!} and {\lstinline!destructor!}, may extend from empty classes (\cref{empty-class}), but not contain any other elements.
```

\item
  An external object class shall be of the specialized class {\lstinline!class!}.
  \begin{nonnormative}

>   This is the only use of {\lstinline!class!} for a non-empty class, \cref{empty-class}.

Reviewer: This sentence appears incomplete.  Did you mean this?
```suggestion
  Apart from empty classes (\cref{empty-class}), this is the only use of {\lstinline!class!}.
```
